inst/qml/auditSelection.qml

```diff
@@ -84,14 +84,27 @@ Form
 		}
 	}
 
-	IntegerField
+	Group
 	{
-		id:										nobs
-		text: 									qsTr("Sample size")
-		name: 									"n"
-		defaultValue: 							0
-		min: 									0
-		fieldWidth: 							60 * preferencesModel.uiScale
+		IntegerField
+		{
+			id:									nobs
+			text: 								qsTr("Sample size")
+			name: 								"n"
+			defaultValue: 						0
+			min: 								0
+		}
+
+		IntegerField
```

Comment:
While this works, there is also the control `SetSeed{}`. You don't want to use that?

Reply:
I saw it comes with a groupbox and a checkbox, but that is not ideal here